lib/datasource/pod/index.ts

```diff
@@ -21,6 +22,13 @@ const cacheMinutes = 30;
 const githubHttp = new GithubHttp(id);
 const http = new Http(id);
 
+const enum URLFormatOptions {
+  WithSharedWithSpec = 0,
+  WithSharedWithoutSpec = 1,
+  WithSpecsWithoutShard = 2,
+  WithoutSpecsWithoutShard = 3,
```

Comment:
```suggestion
  WithSharedWithSpec,
  WithSharedWithoutSpec,
  WithSpecsWithoutShard,
  WithoutSpecsWithoutShard,
```

I think this will work too and use automatical numbering. 😉

Reply:
Ahh thanks for the suggestion.